manifest/manifest.go

```diff
@@ -124,3 +125,9 @@ func ParseDir(chartDir string) ([]*Manifest, error) {
 
 	return files, filepath.Walk(dir, walker)
 }
+
+// IsKeeper returns true if a "helm-keep: true" annotation is present in manifest bytes.
+func IsKeeper(data []byte) bool {
+	var keep = regexp.MustCompile(`\"annotations\": \{(\n.*){0,10}\"helm-keep\": true`)
```

Comment:
You mean if there's a commented-out line like `# "helm-keep": true`? Since we are examining only the reconstituted JSON bytes of the manifest in this case, I don't think that will occur.

But it is a sloppy regex, mostly intentionally so. It says "if you see `"helm-keep": true` in the ten lines following `"annotations":`, assume it's a keeper. I didn't want to get into walking the actual structured data, so searching the byte stream seemed like a reasonable approach.I would love suggestions for how to tighten up the regex.